packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -524,27 +525,8 @@ const mmtParams = ref<MiraTemplateParams>({});
 const configuredMmt = ref(makeConfiguredMMT(mmt.value, knobs.value.transientModelConfig));
 
 const calendarSettings = computed(() => {
```

Comment:
I wonder if this can just be a ref that is assigned after we get the model. It doesn't look like the `model.value` is going to change otherwise.

Reply:
Yea Ill look into making it a `ref`

Reply:
done!